src/compiler/python_generator.cc

```diff
@@ -101,6 +101,11 @@ PrivateGenerator::PrivateGenerator(const GeneratorConfiguration& config,
 void PrivateGenerator::PrintAllComments(StringVector comments,
                                         grpc_generator::Printer* out) {
   if (comments.empty()) {
+    // It's important we print something here and not omit this line,
+    // because we happen to be dealing with an empty construct and
+    // we do not generate "pass" or a comment here, our generated
+    // python code would be invalid.
+    out->Print("pass  # no docstring in proto file");
```

Comment:
"docstring" (or is it "doc string"?) is a Python-specific term, so whatever is missing from the .proto file it's probably called a "service specification comment" or "protodoc" or... I really have no idea. Find out what it is and refer to it by its proper name?